### PR TITLE
220808 add appbar

### PR DESCRIPTION
**Pull request by hayeonhee**
## Changes
- Add a logo image for appbar 
- Add an appbar 
image
image

### CODE REVIEW

**Comment by sebatyler** [comment]
Logo image looks low quality. What about exporting it as 4x in Figma?

**Comment by sebatyler** [comment]
Can you share a screenshot with the new image?

**Comment by hayeonhee** [comment]
I replaced the image. looks better. 👀
image
image

**Comment by hayeonhee** [comment]
Oh, there is a typo in commit message. 😭

**Comment by sebatyler** [comment]
No worries. Not a big deal

File: lib/main.dart [hunk 1]
```diff
@@ -26,19 +26,30 @@ class MyHomePage extends StatelessWidget {
...
-        fontWeight: FontWeight.w400,
-        fontFamily: '-apple-system',
+    return Scaffold(
+      appBar: AppBar(
+        backgroundColor: const Color(0xff040724),
+        toolbarHeight: 72.0,
+        leadingWidth: 86.0,
+        leading: Padding(
+          padding: const EdgeInsets.only(left: 16.0),
+          child: Image.asset('images/logo.png'),
```
Comment (sebatyler) [inline lib/main.dart]: What about setting filterQuality to FilterQuality.high?
When I tested it, it shows better quality.

Comment (hayeonhee) [reply lib/main.dart]: It makes quite a difference. 
 
image

Comment (hayeonhee) [reply lib/main.dart]: I'll setting FilterQuality.high. 
Thanks, I didn't notice 👍

Comment (sebatyler) [reply lib/main.dart]: OK. As soon as you push the commit, I'll approve this PR.

**Review by sebatyler — APPROVED** [review]
Nice work!

**Comment by hayeonhee** [comment]
Thanks for your approval. 😁

**Pull request merged**